.gitignore

```
@@ -1,12 +1,13 @@
 node_modules
 log.json
 /dist
+
 # Hardhat files
 /cache
 /artifacts
 
-# Holds information about the current integration
 /integration-info.json
+/verify
 
```

The depolyments folder is another to ignore after walking through the process.
```suggestion
/deployments

```

---

I want the deployments folder to be present. After someone deploys and verifyies all the contracts on a chain they should make a PR with the deployments file

---

Ah ok, makes sense.